test/integration/simple_operations_test.lua

```diff
@@ -391,3 +391,233 @@ pgroup:add('test_upsert', function(g)
     t.assert_equals(err, nil)
     t.assert_equals(result.rows, {{67, 1143, 'Mikhail Saltykov-Shchedrin', 63}})
 end)
+
+pgroup:add('test_get_partial_result', function(g)
+    -- insert_object
+    local result, err = g.cluster.main_server.net_box:call(
+            'crud.insert_object', {'customers', {id = 1, name = 'Elizabeth', age = 24}})
```

Comment:
What about "bad input" tests? What if user specified some bad field name?

Reply:
> What about "bad input" tests? What if user specified some bad field name?

I believe that we should do as Tarantool:
```lua
tarantool> box.tuple.new({1, {a = {b = 'c'}}})
---
- [1, {'a': {'b': 'c'}}]
...

tarantool> box.tuple.new({1, {a = {b = 'c'}}})['[1]']
---
- 1
...

tarantool> box.tuple.new({1, {a = {b = 'c'}}})['[2]']
---
- {'a': {'b': 'c'}}
...

tarantool> box.tuple.new({1, {a = {b = 'c'}}})['[2].a.b']
---
- c
...
tarantool> box.tuple.new({1, {a = {b = 'c'}}})['K']
---
- null
...
```

This makes this feature quite powerful and allows to fetch not some tuple fields but some nested structures in tuple fields.

Reply:
Issue #120 was opened